nc-news/src/HomeComponents/ArticleList.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import ArticleCard from "./ArticleCard";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getArticles().then((articles) => {
+      setArticles(articles);
```

Comment:
Please add some isLoading functionality so that people can see that the fetch is underway :)